### DISCUSSION TITLE
Assinatura contrato - Data Rudder

### DISCUSSION THREAD

**Discussion by PySulBR** [apyb/comunidade#112] [Geral]
Olá pessoal, tudo bem?
A Data Rudder está precisando de uma pessoa da APyB para assinar o contrato de patrocinio em nome da APyB. Me ofereci para assinar, pois sou do conselho, porém não há nenhum lugar que deixa explicito que posso assinar em nome da APyB.
Qual seria a melhor pessoa para assinar esse documento?

Agradeço desde já,
Talita Rossari.

**Comment by anicelysantos** [comment]
Oi, Talita!

Primeiro é importante que o evento esteja cadastrado para poder utilizar a infraestrutura da ApyB. Sugerimos a leitura deste manual: https://apyb.python.org.br/guias/como-organizar-eventos/

Uma vez cadastrado, a pessoa responsável pelo evento pode assinar, desde que seja membro, assim como você também por ter esse vínculo. Caso o contratante exija que seja uma assinatura da pessoa representante da ApyB, aí nós assinamos.

**Comment by PySulBR** [comment]
Opa @anicelysantos, obrigada pela resposta.
É essa a situação, a empresa exigiu que a assinatura fosse de uma pessoa representante da APyB, como podemos solicitar da APyB essa demanda?

Talita Rossari.

**Reply by anicelysantos** [reply]
Manda em diretoria@python.org.br que assino, mas só posso fazer isso depois que o evento estiver cadastrado, segundo direciona o manual que te enviei. Eventos não cadastrados não possuem relação com a ApyB.

**Reply by PySulBR** [reply]
Perfeito!
tinha pensado que já havia feito o cadastro, mas acho que não cliquei em enviar!
Mil perdões.

Abraços,
Talita Rossari.
